### PR TITLE
VCT integration (uses add-vc)

### PR DESCRIPTION
**Pull request by soluchok**
Signed-off-by: Andrii Soluk <isoluchok@gmail.com>

### CODE REVIEW

File: cmd/orb-server/startcmd/start.go [hunk 1]
```diff
@@ -625,16 +625,10 @@ func (m mockTxnProvider) RegisterForDID() <-chan []string {
...
- ],
- "mockProof": {
-   "type": "JsonWebSignature2020",
-   "proofPurpose": "assertionMethod",
-   "created": "2021-01-27T09:30:15Z",
-   "verificationMethod": "did:example:abcd#key",
-   "domain": "https://witness1.example.com/ledgers/maple2021",
-   "jws": "eyJ..."
- }
+   "algorithm":{
```
Comment (troyronda) [inline cmd/orb-server/startcmd/start.go:628]: The witness need to return an LD proof in activitypub. Where is this mock being used?

**Review by fqutishat — DISMISSED** [review]

File: cmd/orb-server/startcmd/start.go (hunk 1 above)
Comment (soluchok) [reply cmd/orb-server/startcmd/start.go:628]: @troyronda we need to discuss this part. 
`Where is this mock being used?` - currently, we are not using it.

**Review by bstasyszyn — APPROVED** [review]

**Review by sandrask — APPROVED** [review]

File: test/bdd/bddtests_test.go [hunk 2]
```diff
@@ -41,6 +46,42 @@ func TestMain(m *testing.M) {
...
+		s.BeforeSuite(func() {
+			if compose {
+				if err := context.Composition().Up(); err != nil {
+					panic(fmt.Sprintf("Error composing system in BDD context: %s", err))
+				}
+			}
+		})
+
+		s.BeforeSuite(func() {
+			logger.Infof("Running %s", strings.Join(createTree, " "))
```
Comment (fqutishat) [inline test/bdd/bddtests_test.go:58]: why need this logic in BeforeSuite

Comment (fqutishat) [reply test/bdd/bddtests_test.go:58]: you need to move it to bdd step

File: test/bdd/Makefile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Copyright SecureKey Technologies Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+GOBIN_PATH=$(abspath .)/.build/bin
```
Comment (fqutishat) [inline test/bdd/Makefile:5]: why we need make file it's look like shell script

**Review by fqutishat — APPROVED** [review]

**Review by sandrask — APPROVED** [review]

**Pull request merged**